### ISSUE TITLE
Energy tab: home consumption is missing 

### ISSUE THREAD

**Issue by alechenne** [DiedB/Homey-TeslaEnergy#7]
Dear Support,

thanks for upgrading your app to the new SDK3 for Homey Pro 2023. IT works fine.

my only concern is that the energy consumption is missing in the Energy tab (summary of all devices energy) of the Homey App. 

Kind regards

**Comment by petebitt99** [comment]
Hi alechenne. Would you be able to help me with issue 3 below seeing you have been able to successfully connect to your Tesla Gateway.  I am obviously missing something and haven't been able to get any response to my post on the Homey Community forum.

**Comment by alechenne** [comment]
Hi @petebitt99, sure if I can help you, I’ll do it.

I’m not sure to which issue you are talking: the issue #3 I opened is closed. Is it that one?

Currently I migrated today from Homey 2019 to Homey 2023 / 10.0.0rc99. My Powerwall device was migrated / automatically and has worked without to do anything. The Tesla Energy app version is 1.1.4 Experimental.

Please describe me more precisely what is your problem.

PS. It’ 1am44 in Switzerland, I’ll go sleeping. I’ll read your reply tomorrow and reply to it tomorrow evening.

**Comment by petebitt99** [comment]
You are right, it is issue #6 . I think you may have identified my problem. I probably installed the wrong version of the Tesla Energy App thinking that if it installed then it was compatible.  But please read my issue #6 to see what I’m having problems with just in case I’m missing something else.
I will go away to see how to install the experimental version.

**Comment by jonlanceley** [comment]
Hi as @alechenne suggested seeing the powerwall energy data in homey pro would be a great feature to add.

e.g so it appears in here:
https://support.homey.app/hc/en-us/articles/360010187820-Saving-energy-with-Homey-Energy

**Comment by jonlanceley** [comment]
I will answer my own question.  You cant unless you create virtual devices one for each value from the powerwall device.

You can then set these virtual devices so they populate homey energy e.g
![Screenshot_20240115_225101_Homey](https://github.com/DiedB/Homey-TeslaEnergy/assets/4244440/18f6bb25-1fce-43b9-87db-c0267c8908d8)


![20240115_212159](https://github.com/DiedB/Homey-TeslaEnergy/assets/4244440/3f28bbf1-71cf-4c5d-982f-512fd3175bcb)

I used the device capabilities app

Solar and battery configured like this
![Screenshot_20240115_224033_Homey](https://github.com/DiedB/Homey-TeslaEnergy/assets/4244440/c1b09fae-f9a8-4c6e-9895-c25c0d1b26f9)

![Screenshot_20240115_224059_Homey](https://github.com/DiedB/Homey-TeslaEnergy/assets/4244440/732ec42c-c583-49d6-b023-1d7d628c3312)

Home use configured as
![Screenshot_20240115_224659_Homey](https://github.com/DiedB/Homey-TeslaEnergy/assets/4244440/3e21054d-1cf9-4b8c-9898-293e13c59794)

![Screenshot_20240115_224726_Homey](https://github.com/DiedB/Homey-TeslaEnergy/assets/4244440/c1f54c7c-f151-4bc2-b619-a1f6a9d882dd)

Grid use is also class: Other default
![Screenshot_20240115_224928_Homey](https://github.com/DiedB/Homey-TeslaEnergy/assets/4244440/ec19ce77-4a0c-4c44-ab65-ad4477402b04)


And setup a simple flow like this for all 4.
![Screenshot_20240115_223711_Homey](https://github.com/DiedB/Homey-TeslaEnergy/assets/4244440/508e0799-c052-4d95-8d68-94b3cb856f6a)


Note unless i did something wrong there is a bug in homey energy which ive reported which adds solar panel entries to the house total.  So to counter this ive added 1 more virtual device which subtracts it using an advanced flow. Which works most of the time but im still trying to sort out a bug in the flow.

Cc @alechenne
